src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -119,7 +119,7 @@ public function getRules(): array
             'single_line_comment_style' => true,
             'single_line_empty_body' => true,
             'single_line_throw' => false,
-            'string_implicit_backslashes' => ['single_quoted' => 'ignore'],
+            'string_implicit_backslashes' => true,
```

Comment:
for changing any ruleset, remember to update ruleset integration test

Reply:
There is not such test for `@PhpCsFixer` - https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/tree/master/tests/Fixtures/Integration/set

Reply:
@mvorisek it seems like you need to update test case by adding it 😆 (jk).

Reply:
done